examples/design-system-tutorial/src/main.ts

```diff
@@ -0,0 +1,134 @@
+    fluentCard,
+    fluentTreeItem,
+    fluentTreeView,
+    neutralLayerFloatingRecipe,
+    neutralPalette,
+    PaletteRGB,
+    provideFluentDesignSystem,
+    StandardLuminance,
+    SwatchRGB,
+} from "@fluentui/web-components";
+import { parseColorHexRGB } from "@microsoft/fast-colors";
+import { html } from "@microsoft/fast-element";
+import { demoCard, fluentAttribution, Navigation } from "./components";
+
+/**
+ * This expression prevents tree-shaking. See the {@link Navigation} definition
+ * for more information.
+ */
+Navigation;
+
+provideFluentDesignSystem().register(
+    fluentButton(),
+    fluentCard(),
+    fluentTreeView(),
+    fluentTreeItem(),
+    fluentAttribution(),
+    demoCard()
+);
+
+const firstCard = document.querySelector("fluent-demo-card") as HTMLElement;
+const containers = document.querySelectorAll(".container") as NodeListOf<HTMLElement>;
+const firstSection = containers[0];
+const secondSection = containers[1];
+
+/**
+ * Demo 1 - Design Token Default Values
+ *
+ * Every token in Fluent and FAST's design system has a default value. They are set
+ * to the system's defaults to start but you can change them by importing the token
+ * and calling its 'withDefault' method. Each token is strongly typed so that you can
+ * only pass valid values.
+ *
+ *  Uncomment the code below to change the default control corner radius to 0.
+ */
+
+// controlCornerRadius.withDefault(0);
+
+/**
+ * Demo 1.5 - Live Design Tokens
+ *
+ * Design tokens don't just represent static values. They are live. You can change a token
+ * at any time during the life of your application.
+ *
+ * Uncomment the code below to see how we can increase the value of the control
+ * corner radius over time.
+ */
+
+// let value = 0;
+// controlCornerRadius.withDefault(value);
+// setInterval(() => controlCornerRadius.withDefault(value += 1), 1000);
+
+/**
+ * Demo 2 - Light Mode and Dark Mode
+ *
+ * The design system's color model is algorithmically based around the concept
+ * of "luminance" and "layers". By adjusting the luminance of the base layer,
+ * you can switch the design system between light and dark mode, and
+ * anywhere in between (luminance is a value from 0 to 1). To simplify the most common
+ * luminosity settings, the StandardLuminance helper is provided.
+ *
+ * Uncomment the code below to switch the design system into dark mode.
+ */
+
+// baseLayerLuminance.withDefault(StandardLuminance.DarkMode);
+
+/**
+ * Demo 3 - Token Values for DOM Trees
+ *
+ * In addition to being able to set default values for design tokens, you
+ * can also set a token value so that it only applies to a specific part of
+ * the DOM. Use the setValueFor method of a token to set its value only for the
+ * DOM node provided and its descendants.
+ *
+ * Uncomment the code below to set the fill color only for the first card.
+ */
+
+// fillColor.setValueFor(firstCard, (target: HTMLElement) => neutralPalette.getValueFor(target).get(73));
```

Comment:
We should avoid palette indexes. How about one of the neutralLayer recipes instead.

Reply:
I think this example is primarily about illustrating the basic principles of how these design tokens work, and plugging in recipes obfuscates what is happening. I'm going to leave this as-is (but change it to index a light color now that this is fast-based and not fluent).